spec/disable-spec.js

```diff
@@ -1,128 +1,106 @@
-/*
- * decaffeinate suggestions:
- * DS102: Remove unnecessary code created because of implicit returns
- * Full docs: https://github.com/decaffeinate/decaffeinate/blob/main/docs/suggestions.md
- */
 const fs = require('fs-plus');
 const wrench = require('wrench');
 const path = require('path');
 const temp = require('temp');
 const CSON = require('season');
-
 const apm = require('../lib/apm-cli');
 
-describe('apm disable', function() {
-  beforeEach(function() {
+describe('apm disable', () => {
+  beforeEach(() => {
     silenceOutput();
-    return spyOnToken();
+    spyOnToken();
   });
 
-  it('disables an enabled package', function() {
+  it('disables an enabled package', () => {
     const atomHome = temp.mkdirSync('apm-home-dir-');
     process.env.ATOM_HOME = atomHome;
     const callback = jasmine.createSpy('callback');
     const configFilePath = path.join(atomHome, 'config.cson');
-
-    CSON.writeFileSync(configFilePath, { '*': {
-      core: {
-        disabledPackages: [
-          "test-module"
-        ]
+    CSON.writeFileSync(configFilePath, {
+      '*': {
+        core: {
+          disabledPackages: ['test-module']
+        }
       }
-    }
-  });
-
+    });
```

Comment:
I know this has been done in the previous PRs, but as a personal preference I don't like stripping out all the blank lines. I feel that breaking up separate ideas in the code with blank lines helps with readability when reading the code over for the first time, which IMO is the most important read of any code you will do, reading it over the first time around.

I wonder if @confused-Techie has any thoughts on this? If they don't outright feel the same way about it then I might just leave this as-is for consistency with the previous merged PRs.

At least knowing that we committed the machine decaf first, we can go back and see what exactly changed compared to the machine decaf. If I feel really strongly about it in the future I can go back and re-add the blank lines.

So as I said if @confused-Techie or another reviewer agrees with leaving the blank lines then we can standardize on that going forward, otherwise don't worry about it.